tests/ltp/patches/getsockname01.patch

```diff
@@ -0,0 +1,37 @@
+diff --git a/testcases/kernel/syscalls/getsockname/getsockname01.c b/testcases/kernel/syscalls/getsockname/getsockname01.c
+index cce1543fd..060026965 100644
+--- a/testcases/kernel/syscalls/getsockname/getsockname01.c
++++ b/testcases/kernel/syscalls/getsockname/getsockname01.c
+@@ -39,6 +39,10 @@
+  *  None.
+  */
+ 
++/* Patch to remove the EFAULT related sub test as it currently causes
```

Comment:
@ashokchandran , Please put the patch description at the top before diff line (Applies for all patches)